Dockerfile.jdk8

```diff
@@ -54,10 +54,7 @@ EXPOSE 8080 8009
 
 ENV JAVA_OPTS=-Xmx2000m
 
-RUN ln -s $DSPACE_INSTALL/webapps/solr          /usr/local/tomcat/webapps/solr    && \
+RUN mv /usr/local/tomcat/webapps/ROOT /usr/local/tomcat/webapps/ROOT.bk && \
+    ln -s $DSPACE_INSTALL/webapps/spring-rest   /usr/local/tomcat/webapps/ROOT && \
     ln -s $DSPACE_INSTALL/webapps/spring-rest   /usr/local/tomcat/webapps/spring-rest   && \
-    ln -s $DSPACE_INSTALL/webapps/rest          /usr/local/tomcat/webapps/rest    && \
-    ln -s $DSPACE_INSTALL/webapps/oai           /usr/local/tomcat/webapps/oai     && \
-    ln -s $DSPACE_INSTALL/webapps/rdf           /usr/local/tomcat/webapps/rdf     && \
-    ln -s $DSPACE_INSTALL/webapps/sword         /usr/local/tomcat/webapps/sword   && \
-    ln -s $DSPACE_INSTALL/webapps/swordv2       /usr/local/tomcat/webapps/swordv2
+    ln -s $DSPACE_INSTALL/webapps/rest          /usr/local/tomcat/webapps/rest
```

Comment:
@terrywbrady : silly question perhaps, but why are we deploying the `spring-rest` webapp *both* to `ROOT` and `/spring-rest`?  Should it just be deployed to `ROOT`, or is there a reason for both?  This occurs in both Dockerfiles

Reply:
@tdonohue , it makes it easier to test both paths.  If I had to pick just one, I would pick ROOT.

Should I go with that change?

Reply:
Oh, I understand now. If it's easier for now to ensure the "one webapp" is working properly for *both* `ROOT` and subpath deployment, that's fine by me.  Eventually though, we should change this to just *one* path though (ROOT is fine, or if there's a way to make it configurable, that's good too).

Longer term, I'm slightly worried that having it deployed on two paths simultaneously could result in testing oddities -- especially since the follow-up PR to change to a single `dspace.server.url` configuration needs to be well tested to ensure everything is hitting a *single* path.  I don't want something to "appear" to work fine in Docker, when it actually is broken in real life (if for example, I neglect to change a config that is somehow hardcoded to hitting `/spring-rest` instead of `ROOT`).  It's a hopefully unlikely scenario, but still something we should avoid.

Reply:
@tdonohue,

I am still deciding if this flexibility is worthwhile or if it is confusing...
- You could merge as is (with support for both)
- The simplest option would be to just support ROOT
- If we want to support either ROOT or /spring-rest (but not both), I would want to build 2 separate images with clear tags.

Reply:
Ok, let's merge as-is for now.  We should clean up to only use ROOT in the future (perhaps for the next PR)